packages/cli/test/integration/users.controller.test.ts

```diff
@@ -0,0 +1,248 @@
+import * as testDb from './shared/testDb';
+import { setupTestServer } from './shared/utils/';
+import type { User } from '@/databases/entities/User';
+import type { PublicUser } from '@/Interfaces';
+
+const { any } = expect;
+
+const testServer = setupTestServer({ endpointGroups: ['users'] });
+
+let owner: User;
+let member: User;
+
+beforeEach(async () => {
+	await testDb.truncate(['User']);
+	owner = await testDb.createOwner();
+	member = await testDb.createMember();
+});
+
+const validatePublicUser = (user: PublicUser) => {
+	expect(typeof user.id).toBe('string');
+	expect(user.email).toBeDefined();
+	expect(user.firstName).toBeDefined();
+	expect(user.lastName).toBeDefined();
+	expect(typeof user.isOwner).toBe('boolean');
+	expect(user.isPending).toBe(false);
+	expect(user.signInType).toBe('email');
+	expect(user.settings).toBe(null);
+	expect(user.personalizationAnswers).toBeNull();
+	expect(user.password).toBeUndefined();
+	expect(user.globalRole).toBeDefined();
+};
+
+describe('GET /users', () => {
+	test('should return all users', async () => {
```

Comment:
One thing that might be worth adding is a test for retrieving users with non-owners so we can make sure it always returns, since the `GET` endpoint should be public.